src/_includes/partials/global/header.njk

```diff
@@ -0,0 +1,34 @@
+<div class="bg-indigo-800">
+    {% uioTemplate %}
+    {% uioInit %}
+    <div class="header large-viewport">
+        <div class="header__intro">
+            An <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a> Project
+        </div>
+        <div class="header__settings">
+            <ul class="header__language" role="list">
+                <li>
+                    <a href="{% if page.url == '/' %} /en/ {% else %} {{ page.url | replace("/fr/", "/en/") }} {% endif %}" lang="en-CA" hreflang="en-CA"{% if page.lang == 'en' %} aria-current="page"{% endif %}>English</a>
```

Comment:
Much simpler to use the `locale_url` filter: https://www.11ty.dev/docs/plugins/i18n/#locale_url-filter